lib/cli/src/frameworks/vue3/Page.stories.js

```diff
@@ -18,6 +24,4 @@ LoggedIn.args = {
 };
 
 export const LoggedOut = Template.bind({});
-LoggedOut.args = {
-  ...HeaderStories.LoggedOut.args,
-};
+LoggedOut.args = {};
```

Comment:
```suggestion
LoggedOut.args = {
  ...HeaderStories.LoggedOut.args
};
```

This illustrates cascading args -- if we change the implementation of the `Header` object, the page stories don't necessarily have to be updated